.github/scripts/run-benchmarks-remote.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+# Run benchmarks on the remote side
+# Assumes the deephaven-benchmark-*.jar artifact has been built and placed
+
+set -e
+HOST=`hostname`
+RUN_DIR=/root/run
+DEEPHAVEN_DIR=/root/deephaven
+
+if [ ! -d "${RUN_DIR}" ]; then
+  echo "$0: Missing the Benchmark run directory"
+  exit 1
+fi
+
+title () { echo; echo $1; }
+
+title "- Running Remote Benchmark Artifact on ${HOST} -"
+
+title "-- Setting up for Benchmark Run --"
+
+cd ${DEEPHAVEN_DIR};
+docker-compose down
```

Comment:
Any new code we write should be targeted towards `docker compose ...` instead of `docker-compose`.

> From the end of June 2023 Compose V1 won’t be supported anymore and will be removed from all Docker Desktop versions.

> Make sure you switch to [Compose V2](https://docs.docker.com/compose/compose-file/) with the docker compose CLI plugin or by activating the Use Docker Compose V2 setting in Docker Desktop. For more information, see the [Evolution of Compose](https://docs.docker.com/compose/compose-v2/)

https://docs.docker.com/compose/compose-v2/

Reply:
Just tried "docker compose up" and it works on the server.